### PR TITLE
Adds tests for models

### PR DESCRIPTION
**Pull request by stefanjshepard**
This branch sets up 6 rspec tests for our models, adds 2 gems to Gemfile.

Changes:

- spec/models now has 6 _spec.rb tests for the models.  They have the framework for RSpec describe <Object> do.
- rails_helper updated with gems (rspec-rails and simplecov) and rails_helper added to all tests.
- all models now have one-to-many relationships

### CODE REVIEW

File: app/models/invoice.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class Invoice < ApplicationRecord
     belongs_to :merchant
+    belongs_to :customer
     has_many :transactions
     has_many :invoice_items
-    has_many :items
+    has_many :items, through: :invoice_items
```
Comment (jchirch) [inline app/models/invoice.rb:6]: I'm looking forward to this learning opportunity on the concept of many-to-many relationships

**Comment by reneemes** [comment]
Thank you for adding these relationships. We went over these on zoom and we decided we will add more later if necessary.

File: spec/models/item_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
```
Comment (jchirch) [inline spec/models/item_spec.rb:1]: Good outline, will fill out this week.

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -0,0 +1,70 @@
+require "simplecov"
```
Comment (jchirch) [inline spec/rails_helper.rb:1]: Simplecov is a fantastic resource that will be indispensable in this project

**Pull request merged**